src/firefox-infobar-ui/static/_locales/cs/messages.json

```diff
@@ -1,22 +1,22 @@
 {
   "extensionName": {
-    "message": "Firefoxace",
+    "message": "Překladač pro Firefox",
     "description": "Name of the extension"
   },
   "extensionDescription": {
-    "message": "Neural Machine Překlad pro prohlížeč.",
+    "message": "Strojový překladač cizích jazyků založený na neuronové síti.",
```

Comment:
Here it's the same. "Neuronový strojový překlad v prohlížeči" is a more adequate translation of the original "Neural Machine Translation for the browser."

Reply:
Thank you for the proposal. I updated the PR.